### PR TITLE
feat: Add README in Romanian language

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Proposed changes

- Add README in Romanian language.

## Types of changes
- [ ] Proofreading (fix spelling, grammar, punctuation, accuracy) [Proofreading](https://www.scribbr.com/language-rules/what-is-proofreading/)
- [x] New content (new section, new readme, etc.)
- [ ] Modifications (modifying existing content)
- [ ] Other (please describe)

## Checklist
- [ ] Tested any code that was added/modified in this PR
- [ ] Added external links
- [x] Added internal links
- [ ] Added images
- [ ] Added videos
- [ ] Added code snippets

### Reviewers: 
- [x] @shhossain

### CODE REVIEW

File: README_ro.md [hunk 1]
```diff
@@ -0,0 +1,1188 @@
...
+
+## Ghidul de contribuție
+Dacă sunteți interesat să contribuiți la acest proiect, vă rugăm să acordați un moment pentru a revizui [CONTRIBUTING.md](https://github.com/shhossain/computer_science/blob/main/CONTRIBUTING.md) pentru instrucțiuni detaliate despre cum să începeți. Contribuțiile dumneavoastră sunt foarte apreciate!
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![Toți colaboratorii](https://img.shields.io/badge/all_contributors-132-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+## Cuprins
+- [Introducere](#introduction)
```
Comment (shhossain) [inline README_ro.md:23]: change table of contents links:
[Introducere](#introduction) to [Introducere](#Introducere)

**Review by shhossain — CHANGES_REQUESTED** [review]

File: README_ro.md (hunk 1 above)
Comment (vladimir-cucu) [reply README_ro.md:23]: Thanks for looking over my PR! I'll fix this issue everywhere it occurs.

Comment (vladimir-cucu) [reply README_ro.md]: @shhossain I've fixed the issue and some other issues as well. I think it should be good now!

**Comment by shhossain** [comment]
@all-contributors add @vladimir-cucu for content.

**Pull request merged**